.github/workflows/ci.yml

```diff
@@ -91,6 +91,15 @@ jobs:
           DOCKER_BUILDKIT: 1
         run: docker build -f ./.github/actions/bitcoin-int-tests/Dockerfile.rustfmt .
 
+  rustdoc:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run rustdoc check
+        env:
+          DOCKER_BUILDKIT: 1
+        run: docker build -f ./.github/actions/bitcoin-int-tests/Dockerfile.rustdoc .
```

Comment:
The only thing I would add here is that this use docker's `-o` option to export the data from the dockerfile. It shouldn't require much update here, but it'll require a little change in the dockerfile side as well.

Reply:
We do something similar in the `docs-pr.yml`

Reply:
The initial intent was to use rustdoc to enforce documentation but not to generate or export it. I can follow your suggestion to publish the config doc as well.